### PR TITLE
Add basic popup overlay control

### LINKED ISSUE #49: Create a popup which accepts dynamic content

### PR DESCRIPTION
**Pull request by BAL4-R**
Resolves #49 
Create basic popup overlay control to add dynamic content into a popup overlay

### CODE REVIEW

File: DemoApp/DemoApp/MauiProgram.cs [hunk 1]
```diff
@@ -10,6 +11,7 @@ public static MauiApp CreateMauiApp()
         var builder = MauiApp.CreateBuilder();
         builder
             .UseMauiApp<App>()
+            .ConfigurePopups()
```
Comment (rthanga1) [inline DemoApp/DemoApp/MauiProgram.cs:14]: The entire popup related logic should not be exposed to the clients

File: DemoApp/DemoApp/MainPage.xaml.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿using Trimble.Modus.Components;
+﻿using Trimble.Modus.Components.Overlay.Services;
```
Comment (rthanga1) [inline DemoApp/DemoApp/MainPage.xaml.cs:1]: Please rename 'Overlay' namespace into 'Popup'

**Review by rthanga1 — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Controls/Overlay/Animations/FadeAnimation.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
+﻿using Trimble.Modus.Components.Overlay.Animations.Base;
+using Trimble.Modus.Components.Overlay.Pages;
+
+namespace Trimble.Modus.Components.Overlay.Animations;
+
+public class FadeAnimation : BaseAnimation
```
[the remark this answers is not in the document]
Comment (BAL4-R) [reply Trimble.Modus.Components/Controls/Overlay/Animations/FadeAnimation.cs]: Done

File: DemoApp/DemoApp/MainPage.xaml.cs (hunk 2 above)
Comment (BAL4-R) [reply DemoApp/DemoApp/MainPage.xaml.cs]: Done

**Review by rthanga1 — APPROVED** [review]

File: .github/workflows/ci.yaml
Comment (rthanga1) [inline .github/workflows/ci.yaml:1]: @jeffdoolittle I made some change to the ci.yaml file. There's still a lot to do, but this fixes the build failures. Please go over my changes and let me know if this is okay.

**Pull request merged**